package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "devDependencies": {
+    "@babel/cli": "^7.15.7",
+    "@babel/core": "^7.15.8",
+    "@babel/preset-env": "^7.15.8",
+    "@babel/preset-react": "^7.14.5",
+    "@babel/preset-typescript": "^7.15.0",
+    "@testing-library/react": "^12.1.2",
+    "eslint": "^8.0.1",
+    "jest": "^27.2.5",
+    "prettier": "^2.4.1",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "jsdom": "^18.0.0"
+  },
+  "peerDependencies": {
+    "react": "^17.0.2"
```

Comment:
Does this library require react 17, would `^16 || ^17` work here to support react 16 apps using the hook?

Reply:
Both should work